game/src/main/scala/io/aigar/controller/GameController.scala

```diff
@@ -1,31 +1,22 @@
 package io.aigar.controller
 
+import io.aigar.game._
 import io.aigar.controller.response._
 import org.json4s.{DefaultFormats, Formats, MappingException}
 import org.scalatra.json._
+import scala.util.Try
 
-class GameController extends AigarStack with JacksonJsonSupport {
-  object GameStates {
-    var all = List(
-      GameState(
-        1,
-        5,
-        List(
-          Player(12, "such", 555, List(Cell(5, 5, Position(10,10), Position(10, 10)))),
-          Player(13, "wow", 555, List[Cell]())
-        ),
-        Food(List(Position(5,5)), List[Position](), List[Position]()),
-        Dimensions(10, 10),
-        List[Position]()
-      )
-    )
-  }
+class GameController(val game: GameThread)
```

Comment:
Why `val`? Not sure to understand http://stackoverflow.com/questions/14694712/do-scala-constructor-parameters-default-to-private-val